ods_ci/tests/Tests/100__deploy/130__operators/150__dsp_operator/150__dsp_operator.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Documentation       Test Cases to verify Serverless installation
+
+Library             Collections
+Library             SeleniumLibrary
+Library             OpenShiftLibrary
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource            ../../../../Resources/OCP.resource
+Resource            ../../../../Resources/RHOSi.resource
+
+Suite Setup         Suite Setup
+Suite Teardown      Suite Teardown
+
+
+*** Test Cases ***
+Detect Pre-existing Install Of Argo Workflows And Block RHOAI Install
+    [Documentation]    Detect Pre-existing Install Of Argo Workflows And Block RHOAI Install
+    [Tags]                  Operator                ODS-2651                Tier1
+    ${return_code}          ${output}               Run And Return Rc And Output
+    ...                     oc apply -f ods_ci/tests/Resources/Files/argo/crd.workflows.yaml
+    Log To Console          ${output}
+    Should Be Equal As Integers
+    ...                     ${return_code}
+    ...                     0
+    ...                     msg=Error while applying the provided file
+    Open Installed Operators Page
+    Navigate to Installed Operators
+    ${is_operator_installed}                        Is Operator Installed                           ${OPERATOR_NAME}
+    IF    ${is_operator_installed}    Uninstall ODH Operator
+    ODH Operator Should Be Uninstalled
+    Open OperatorHub
+    Install ODH Operator
```

Comment:
why?
I think it'd safer not to put a such test in Tier1.
Also, it says "ODH", does it do the same for RHOAI?

Reply:
We are leaving tests on Tier1 by default, the name ODH in that keyword already existed so I would have to refactor it in order to change name

Reply:
> We are leaving tests on Tier1 by default


I'm not sure is a good reason for adding a potential destructive test in Tier1

Reply:
on which Tier do you suggest to set it?